xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/edit.less

```diff
@@ -22,11 +22,7 @@
   }
 
   input[type="checkbox"] {
-    vertical-align: middle;
-    position: relative;
-    margin: 0;
     margin-left: 1em;
```

Comment:
Not sure why you left this property and didn't move everything.

Reply:
@evalica:
This property is used to have a gap between the checkboxes and other inputs.
If I also move this property to be used gloabally, every checkboxes will then be affected. If you have a checkbox alone on a line, it will seem to be indented.
Maybe a better solution could be not to put a margin to the left of the checkbox but a margin to the right of the labels and inputs next to it, but we have to decide if it should be specific to the "save bar" or to every inputs.

Reply:
k